src/tests/Microsoft.PowerFx.Interpreter.Tests/ConfigTests.cs

```diff
@@ -0,0 +1,476 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Binding.BindInfo;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Types.Enums;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Intellisense;
+using Microsoft.PowerFx.Tests.IntellisenseTests;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class ConfigTests
+    {
+        [Fact]
+        public async Task BasicVariables()
+        {
+            // Share a config
+            var s1 = new SymbolTable();
+            s1.AddVariable("local", FormulaType.Number);
+
+            // Per expression.
+            var engine = new RecalcEngine();
+            engine.UpdateVariable("global", 2.0);
+
+            var runtimeConfig = new RuntimeConfig();
+            runtimeConfig.SymbolValues["local"] = FormulaValue.New(3);
+
```

Comment:
It feels a little disconnected to have the symbol table nd the runtime config's symbol values separately. I hope we can find a better way to do it than using variable names as the "join key" between the type (in the symbol table) and the runtime value (in the runtime config).

Reply:
Agreed, but the problem is how do we
- declare something at bind time 
- and then eval it many times, passing in different parameters for each eval. 

Wondering if there's something elegant we can do for interpreter cases (where the binder has the runtime values), like have a runtime config double as a symbol table.

Reply:
Latest iterations have some improvements here.